docs/source/en/api/normalization.md

```diff
@@ -29,3 +29,19 @@ Customized normalization layers for supporting various models in 🤗 Diffusers.
 ## AdaGroupNorm
 
 [[autodoc]] models.normalization.AdaGroupNorm
+
+## AdaLayerNormContinuous
+
+[[autodoc]] models.normalization.AdaLayerNormContinuous
+
+## LayerNorm
+
+[[autodoc]] models.normalization.LayerNorm
```

Comment:
`LayerNorm` is the reason why doc-builder is not building.

I've inspected a bit. 
All other norms are diffuser defined classes:
```
>>> diffusers.models.normalization.GlobalResponseNorm
<class 'diffusers.models.normalization.GlobalResponseNorm'>
```
whereas `LayerNorm` seems to be alias to pytorch defined class:
```
>>> diffusers.models.normalization.LayerNorm
<class 'torch.nn.modules.normalization.LayerNorm'>
```

Therefore, as a simple fix: I'd suggest replacing `[[autodoc]] models.normalization.LayerNorm` with something like: `You can also use LayerNorm and add markdown link to pytorch layernorm doc`

Reply:
Thanks @mishig25. I think it'd be okay to remove LayerNorm from our doc because our implementation is really a special case and is already supported in the latest versions of PyTorch.